src/osemgrep/cli_ci/Ci_subcommand.ml

```diff
@@ -144,42 +145,70 @@ let run (conf : Ci_CLI.conf) : Exit_code.t =
             (* TODO: metadata_dict["is_sca_scan"] = supply_chain *)
             (* TODO: proj_config = ProjectConfig.load_all()
                metadata_dict = {**metadata_dict, **proj_config.to_dict()} *)
-            let scan_id =
+            match
               Scan_helper.start_scan ~dry_run:conf.dryrun ~token
                 Semgrep_envvars.v.semgrep_url metadata_dict
-            in
-            let at_url_maybe ppf () =
```

Comment:
I try to avoid nested function when I can. Maybe this can be move at the toplevel in a  Helpers section,
with a small comment before.

Reply:
done in [66c9692](https://github.com/returntocorp/semgrep/pull/8149/commits/66c9692fe5ec4db227b46887b15e7b18f66cf8e5)